dev/docs/dartdoc_options.yaml

```diff
@@ -0,0 +1,3 @@
+dartdoc:
+  ignore:
+    - no-documentable-libraries
```

Comment:
Are we adding this ignore here instead of in https://github.com/flutter/flutter/blob/master/dartdoc_options.yaml so it only applies to that empty `Flutter` package and nothing else?

Should https://github.com/flutter/flutter/blob/master/dartdoc_options.yaml make this warning fetal (by listing it in the error section) for all other packages?

Reply:
I don't know -- obviously something is very catastrophically wrong in Flutter, or more likely, dartdoc internals, if that happens so it seems fair to add it.   I'll try it and make sure it doesn't have any adverse consequences, then follow up here.

Reply:
Oh, and yes, it is in dartdoc_options.yaml in `dev/docs` to limit the scope of the setting.

Reply:
@goderbauer Added as an error in the top level `dartdoc_options.yaml`; seems to work fine and there aren't any more stub packages hiding out that are impacted.  PTAL.